aclogview/CM_Communication.cs

```diff
@@ -140,6 +146,53 @@ public override void contributeToTreeView(TreeView treeView)
         }
     }*/
 
+    public class Recv_ChatRoomTracker : Message
+    {
+        public uint AllegianceChannel;
+        public uint GeneralChannel;
+        public uint TradeChannel;
+        public uint LFGChannel;
+        public uint RoleplayChannel;
+        public uint Olthoi;
+        public uint Society;
+        public uint Unknown_3;
+        public uint Unknown_4;
+        public uint Unknown_5;

```

Comment:
FYI these unknowns seem to be SocietyCelHan, SocietyEldWeb, and SocietyRadBlo - each corresponding to their respective society.